### DISCUSSION TITLE
Gatsby support?

### DISCUSSION THREAD

**Discussion by edenworky** [react-cosmos/react-cosmos#1627] [Q&A]
What is the status of Gatsby support? React Cosmos looks fantastic, but I can't get it to work for components in Gatsby with the default configuration.

**Comment by ovidiuch** [comment]
@edenworky I know it's been a while but I wanted to give up an update now that v6 is ready to use. Unfortunately I don't have experience with Gatsby so I can't give guidance here. But I added a guide on [integrating Cosmos with any renderer](https://github.com/react-cosmos/react-cosmos/blob/f26189c113b0ed8565f8ac2f5679b507dc5d02a2/docs/getting-started/custom-bundler.md). Maybe someone with Gatsby knowledge can put something together using this guide. Afterwards we can work together to turn it into a proper plugin.
